.gitmodules

```diff
@@ -1,9 +1,3 @@
-[submodule "src/main/cpp/3rdparty/SPIRV-Cross"]
```

Comment:
Are you sure this is unneeded? I thought I was using spirv-cross for shader reflection